test/lib/index.test.ts

```diff
@@ -51,6 +51,7 @@ fixtureTest(
   'CocoaPods’ integration spec install_new',
   'cp-integration-install_new'
 );
+fixtureTest('Parse iina’s Podfile.lock', 'iina');
```

Comment:
Maybe the `description` should mention what is actually under test a little more? Something like:

'Platform specific transitives'